cluster-autoscaler/cloudprovider/aws/README.md

```diff
@@ -48,6 +48,8 @@ by specifying Auto Scaling Group ARNs in the `Resource` list of the policy. More
 information can be found
 [here](https://docs.aws.amazon.com/autoscaling/latest/userguide/control-access-using-iam.html#policy-auto-scaling-resources).
 
+For an example of how to use AWS IAM OIDC with the Cluster Autoscaler please see [here](CA_with_AWS_IAM_OIDC.md).
```

Comment:
Let's have a another header for oidc setup? This will be easier for user to notice this I think?

Reply:
Good idea, done.